frontend/src/components/navbar/NavbarLinks.tsx

```diff
@@ -49,7 +49,7 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   user: {
-    background: "#193c27", //065A5A
+    background: "#065A5A",
```

Comment:
Is this a color in our palette? Otherwise, should it be?

Reply:
Yes, it should be added. I'm not particularly happy with the work I've done on these components